tensorflow/core/kernels/mkl/mkl_quantized_pooling_ops_test.cc

```diff
@@ -125,20 +104,12 @@ TEST_F(QuantizedPoolingTest, SmallAveragePooling) {
   TF_ASSERT_OK(RunOpKernel());
 
   const Tensor& output = *GetOutput(0);
-  Tensor output_quantized;
-  if (!NativeFormatEnabled()) {
-    const Tensor& mkl_shape_tensor = *GetOutput(3);
-    ConvMklToTF conv_comp;
-    conv_comp.ConvertMKL2TF<quint8>(DT_QUINT8, output, mkl_shape_tensor,
-                                    output_quantized);
-  }
-
   const float output_min = GetOutput(1)->flat<float>()(0);
   const float output_max = GetOutput(2)->flat<float>()(0);
-  Tensor output_float = QuantizedTensorToFloat<quint8>(
-      NativeFormatEnabled() ? output : output_quantized, output_min,
-      output_max);
-
+  const Tensor* mkl_shape_tensor =
```

Comment:
Nit: Can we add `_ptr` to the name? Same for line 180.
```suggestion
  const Tensor* mkl_shape_tensor_ptr =
```

Reply:
Sure. Done.